src/agent/onefuzz-agent/src/tasks/coverage/libfuzzer_coverage.rs

```diff
@@ -115,12 +104,11 @@ impl CoverageTask {
         // Update the total with the coverage from each seed corpus.
         for dir in &self.config.readonly_inputs {
             verbose!("recording coverage for {}", dir.path.display());
-            init_dir(&dir.path).await?;
-            sync_remote_dir(&dir, SyncOperation::Pull).await?;
+            dir.init_pull().await?;
             self.record_corpus_coverage(&mut processor, dir).await?;
             fs::remove_dir_all(&dir.path).await?;
-            sync_remote_dir(&self.config.coverage, SyncOperation::Push).await?;
         }
+        self.config.coverage.sync(Push).await?;
```

Comment:
Consider adding/using `sync_push()` and `sync_pull()` methods, maybe only preserving un-suffixed `sync()` if there are cases where we need the direction to be dynamic.

Reply:
(that said, even the change as-is is a great readability improvement, IMO)